### ISSUE TITLE
Unable to see the users and groups on ACL dialog after zooming in twice

### ISSUE THREAD

**Issue by v-xuanzh** [Microsoft/AzureStorageExplorer#1317]
**Storage Explorer Version:** 1.8.0_20190422.2
**Platform/OS:** Linux Ubuntu/macOS High Sierra/Windows 10
**Architecture:** ia32/x64
**Regression From:** Not a regression

**Steps to reproduce:**
1. Zoom in Storage Explorer twice.
2. Expand one ADLS Gen2 account -> Blob Containers.
3. Right click one blob container then select 'Manage Access...'.
3. Check the users and groups.

**Expect Experience:**
Able to see the users and groups.

**Actual Experience:**
Unable to see the users and groups.
![image](https://user-images.githubusercontent.com/34729022/56559875-ba14db80-6557-11e9-90a8-aec60d85088d.png)

**Comment by craxal** [comment]
Closing as Won't Fix. We document that zooming in may require window resizing.
